src/services/http/httpService.js

```
@@ -76,6 +74,16 @@ function filtersToQuery(filters) {
         return response.json();
     },
 
+    getLocationsData : async (lat, lon, filters) => {
```

ESLint

[error] Delete `·`

(prettier/prettier)